src/test/fuzz/fuzz.cpp

```diff
@@ -72,8 +72,8 @@ auto& FuzzTargets()
 
 void FuzzFrameworkRegisterTarget(std::string_view name, TypeTestOneInput target, FuzzTargetOptions opts)
 {
-    const auto it_ins{FuzzTargets().try_emplace(name, FuzzTarget /* temporary can be dropped after clang-16 */ {std::move(target), std::move(opts)})};
-    Assert(it_ins.second);
+    const auto [it, ins]{FuzzTargets().try_emplace(name, FuzzTarget /* temporary can be dropped after Apple-Clang-16 ? */ {std::move(target), std::move(opts)})};
```

Comment:
I can confirm that the GHA [macos-14](https://github.com/actions/runner-images/blob/main/images/macos/macos-14-arm64-Readme.md) image with Xcode 16.0 (beta) (build 16A5171c) does not require this workaround.

Reply:
It did not mean any changes to the current branch :)

It was just a note (mostly for myself).